README.md

```diff
@@ -12,7 +12,19 @@ Performs a migration from CFME 5.7 to 5.8 utilizing steps from [Migrating to Red
 * Disks have already been resized.
 * Environments are not utilizing database replication.
 
+#### Additionaly required playbooks
+* cfme-repositories.yml
```

Comment:
this isn't a playbook its a task list and since it's included as part of this repository i would not explicitly call it out as a dependency since there is nothing the user of this repository would need to do.